mesh/ncmesh.cpp

```diff
@@ -119,6 +120,13 @@ NCMesh::NCMesh(const Mesh *mesh)
       CheckSupportedGeom(geom);
       GI[geom].InitGeom(geom);
 
+      //If we have pyramids we will need tets after refinement
+      if (geom == Geometry::PYRAMID)
+      {
+         CheckSupportedGeom(Geometry::TETRAHEDRON);
```

Comment:
This line is superfluous because `Geometry::TETRAHEDRON` is supported.

Reply:
TetraHedra are supported by default?  If that is the reason I would like to leave this in here in case that ever changes.

Reply:
It seems improbable that the NC tet support could be dropped... do you mean to keep the line for documentation purposes?

Reply:
Oh wait I was misunderstanding the purpose of CheckSupportedGeometry.  Yeah you are right this is superfluous.  I'll drop it in my next push.